MixerDLL/mixer/dllmain.cpp

```diff
@@ -91,17 +91,15 @@ void initAudio(){
 	pDevice->Activate(__uuidof(IAudioEndpointVolume), CLSCTX_ALL, NULL, (VOID**)&pEpVol);
 	// get Audio Session Enumerator
 	pAudioSessionManager2->GetSessionEnumerator(&pAudioSessionEnumerator);
-	
-	pDevice.Release();
-	pDeviceEnumerator.Release();
-	pAudioSessionManager2.Release();
+
+	return true;
 }
 
-char * errorMessage = NULL;
-char * getErrorMessage(){
+char* errorMessage = NULL;
+char* getErrorMessage(){
 	return errorMessage;
 }
-int getSessionCount(){
+int getSessionID(){
```

Comment:
Why did you choose the name `getSessionID`? For me that is confusing becuse ID(*Identity*) has nothing to do with the number of sessions.